scripts/_template_ProcessEnsemblGenomeLoading.slurm.sh

```diff
@@ -1,57 +1,156 @@
 #!/usr/bin/bash
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+####################
+### Create ensembl-mod-env for genome load production processing:
+# Step1: get latest updates to ensembl-mod-env repo ! This is critical to ensure proper functionality. 
+# Step2: Select appropriate python version, this e.g. use relatively latest python version v3.11.7
+  # - module load python/3.11.7
+# Step3: Create production environment, using load_genome.conf (https://github.com/Ensembl/ensembl-mod-env/blob/main/confs/ensembl/load_genome.yml)
+# whilst selecting the base branch to be the same as the ensembl schema version for the core creation [ENS_VERSION]
+  # - E.g. modenv_create -b 114 ensembl/load_genome e114_genomeload
+## Load module env, you will need to do this manually outside of this wrapper
+  # -E.g.  module load ensembl/e114_genomeload
+####################
+
+## set and ENS_VERSION
+if [[ $ENSEMBL_VERSION ]]; then
+  ENS_VERSION=$ENSEMBL_VERSION
+  echo "ENSEMBL VERSION detected: '$ENS_VERSION'"
+else
+  # e.g 111 i.e. the ensembl schema version to use for loading
+  ENS_VERSION=""
+  if [[ -z $ENS_VERSION ]]; then
+    echo "Please define the ENSEMBL VERSION: ENS_VERSION, See $0 (line: 35)"
+    exit 0
+  fi
+fi
+
+##### How to set up the main production environment.
+# METHOD A: Setup when using ensembl-mod-env for production
+# If using mod-env you must ensure the environment variable MODENV_ROOT is defined !
+if [[ $MODENV_ROOT ]] && [[ $MODENV_HOME ]]; then
+  echo "Detecting 'ensembl-mod-env' production environment setup...."
+  PRODUCTION_MODENV_NAME="" #< E.G. NOTE: Ensure MODENV_DIR DOESN'T END WITH: "/"
+  if [[ $PRODUCTION_MODENV_NAME ]]; then
+    MODENV_DIR="${MODENV_HOME}/${PRODUCTION_MODENV_NAME}" ## MODENV_ROOT must be defined and should be if ensembl-mod-env is correctly setup for $USER
+    echo "Using modenv setup ?.... if so its configured to this path: '$MODENV_DIR'"
+    ENS_MAIN_METAZOA_PROD=${MODENV_DIR}/ensembl-production-metazoa
+    ln -s $ENS_MAIN_METAZOA_PROD ./ensembl-production-metazoa
```

Comment:
why do we need this?

Reply:
line 102-121 of mz_generic was prepending pwd path to MZ_CONFIG variable passed to the job submission. It could of been a problem with my env as I was unaware that the ensembl-mod-env ENV variable MODENV_ROOT was not defined properly following an update to my ensembl-mod-env repo. The symlinked folder resolved the issue for me hence I added this here